src/run.ts

```diff
@@ -278,22 +330,31 @@ ${
 
   let searchResult = await searchResultPromise;
   console.log(JSON.stringify(searchResult.data, null, 2));
+
+  let prBody = await getChangesetsMessage({
+    hasPublishScript,
+    preState,
+    branch,
+    changedPackagesInfo,
```

Comment:
If you follow up on the comment [here](https://github.com/changesets/action/pull/157/files#r851184758) then this change will be required too:
```suggestion
    changedPackagesInfo: await changedPackagesInfo,
```